manual/photon.xml.template

```diff
@@ -10,7 +10,7 @@
             <Label>Hostname</Label>
             <Description>Hostname (FQDN) of system</Description>
         </Property>
-        <Property ovf:key="guestinfo.ipaddress" ovf:type="string" ovf:userConfigurable="true">
+        <Property ovf:key="guestinfo.ip_address" ovf:type="string" ovf:userConfigurable="true">
```

Comment:
Just glancing over some of the changes in this PR (not a thorough review) but I was curious on why the OVF keys are being changed? The reason I ask is that there are folks who've come to rely on these for automation/scripting purposes

Reply:
After have a look I don't see any reason it shouldn't work the other way around.
I've just pushed a new commit that reverts the OVF properties back to the current development version and changes the variables throughout the setup scripts and ytt templates to match the OVF properties instead.
Is this a better approach do you think?

Reply:
Yup and I totally get where you were going with the consistency, especially for debugging and for new users. Appreciate the change. I'll find some time to do more thorough review and may have more questions

Reply:
👍 Thank you.

Don't be too thorough with a review just yet mind... there's still a lot to do, but any comments on the overall approach would be great.